src/content/en/tools/service-worker-libraries/index.md

```diff
@@ -3,30 +3,64 @@ book_path: /web/tools/_book.yaml
 description: Service Worker Libraries.
 
 {# wf_published_on: 2015-01-01 #}
-{# wf_updated_on: 2016-09-12 #}
+{# wf_updated_on: 2016-11-03 #}
 
 # Service Worker Libraries {: .page-title }
 
-## What are the Service Worker Libraries?
+<div class="attempt-right">
+  <figure>
+    <img src="../images/tools-landing-page.gif" max-width="50%">
```

Comment:
Did width="80%". 100 covers the page navigation. 80 has appropriate behavior on a variety of device and screen sizes.